atc/db/check_lifecycle.go

```diff
@@ -37,5 +37,14 @@ func (lifecycle *checkLifecycle) RemoveExpiredChecks(recyclePeriod time.Duration
 		RunWith(lifecycle.conn).
 		Exec()
 
-	return err
+	if err != nil {
+		return 0, err
+	}
+
+	affected, err := result.RowsAffected()
+	if err != nil {
+		return 0, err
+	}
+
+	return int(affected), nil
```

Comment:
hey, while it's fine to go from `int` to `int64` as 

>

(see https://golang.org/pkg/builtin/#int)

Reply:
I will have to do the int() convertion either here or when the meter.IncDelta is called [here](https://github.com/concourse/concourse/blob/9b9c15e5af33ab9c4176875e5485b48397b91e12/atc/metric/meter.go#L11)

We are using the IncDelta several places so I would prefer keep it as it is right now and have a seperate issue for int & int64 row affected counts.

Reply:
sgtm! 👍